### DISCUSSION TITLE
HYMAP2 routing resolution

### DISCUSSION THREAD

**Discussion by sclee1717** [NASA-LIS/LISF#1543] [Q&A] [answered]
Hi all,

I'm running Noah-MP4.0.1 with HYMAP2 routing model.

Just one curiosity, will it produce reliable streamflow results when ran with 0.1 deg. resolution (using 0.25 deg. resolution parameter files)?

I found that LDT generates 0.1 deg. lis input file (parameters are generated in 0.1 deg. by nearest interpolation) and LIS also runs without errors but can I believe that this is correct, or do I need to run in a 0.25 deg. resolution?

thanks in ahead!
Lee

**Comment by agetiran** [comment, marked as answer]
Hi @sclee1717,
currently, LDT simply read HyMAP parameter files and write them out in the lis_input.nc file. There's no interpolation, and spatial resolutions of your domain and HyMAP parameters must match. That said, you cannot use 0.25-deg parameters over a 0.1-deg resolution domain.

**Reply by sclee1717** [reply]
Thank you for the answer @agetiran!
